### ISSUE TITLE
Update README with reliable instructions to start Trustify

### ISSUE THREAD

**Issue by carlosthe19916** [guacsec/trustify#373]
I followed the README file and executed `cargo run --bin trustd` and all the api rest endpoints under /api/v1 has security in place (generate 401) error while the UI didn't receive the proper configuration for initializing the Login flow. Could we update the README with proper commands (specially the PM mode please)

**Comment by ctron** [comment]
I think it works as expected. The UI gets initialize with auth enabled. However, the embedded OIDC server does not ask for a password. Still, the UI sends a bearer token to the backend. When using the API directly, this must be sent as well.

**Comment by carlosthe19916** [comment]
Thanks for your reply @ctron 
- I was not aware of the fact that the embedded OIDC does not ask for a password, that explain things. I had a problem with cache but now I can get access to the UI through the server.

I still have one issue, how could I get access to the Swagger UI? I tried http://localhost:8080/openapi and I get the following error:

![Screenshot from 2024-06-05 15-07-52](https://github.com/trustification/trustify/assets/2582866/c79c264c-83b8-4c70-9fdb-6e3d7b8ac288)

I also tried http://localhost:8080/swagger-ui/ as stated in the README but it didn't work either. Which is the proper URL for swagger? Or might it be that is is a bug?
